internal/iptables.go

```diff
@@ -195,11 +195,15 @@ func (b Banner) LogRegexBan(
 		disableLogging = 1
 	}
 
+	// we append "| <http status code>" at the end of banjax-format.log for speical regex rule
+	// here we split | and ignore everything on the right
+	vertical_bar_split := strings.SplitN(words[5], "|", 2)
+
 	logObj := LogJson{
 		words[3], // path
 		timeString,
 		ruleName,
-		words[5], // client_ua
+		strings.TrimSpace(vertical_bar_split[0]), // client_ua
```

Comment:
Something to improve in the future; use constants or enums instead of raw indices to access the String array, e.g. `vertical_bar_split[client_ua]`.